Descriptor: Debian-specific changes due to different supervisor config directory.
Forwarded: not-needed.
--- a/modules/500-start-memcached/run.sh
+++ b/modules/500-start-memcached/run.sh
@@ -6,8 +6,8 @@
 . "`dirname $0`/../compat.sh"
 
 # Démarrage par supervisor plutôt qu'autonome.
-service=supervisord
-cp -a memcached.ini /etc/supervisord.d/
+service=supervisor
+cp -a memcached.ini /etc/supervisor/conf.d/
 change_svc memcached off
 change_svc $service on
 service $service status &> /dev/null
